examples/platforms/cc2538/radio.c

```diff
@@ -300,7 +300,7 @@ RadioPacket *otPlatRadioGetTransmitBuffer(void)
     return &sTransmitFrame;
 }
 
-int8_t otPlatRadioGetNoiseFloor(void)
+int8_t otPlatRadioGetRssi(void)
 {
     return 0;
```

Comment:
If you aren't returning the actual value, return 127 instead, to indicate that the returned value is invalid.

We may want to add a constant for it though. `kOtUndefinedPower`, perhaps?